app/src/main/java/org/oppia/app/settings/profile/ProfileDeletionDialogFragment.kt

```diff
@@ -0,0 +1,7 @@
+package org.oppia.app.settings.profile
```

Comment:
If we aren't using this in this PR, can we remove it and add it later?

Reply:
Removed.